docs/contributing-to-airbyte/README.md

```diff
@@ -108,6 +108,35 @@ Trying to contribute, and don't see the change you want to make in this list? Ca
 ### Airbyte CI workflows
 * [Testing by SonarQube](sonar-qube-workflow.md)
 
+## Breaking Changes to Connectors
+
+Often times, changes to connectors can be made without impacting the user experience.  However, there are some changes that will require users to take action before they can continue to sync data.  These changes are considered **Breaking Changes** and require a
+
+1. A **Major Version** increase 
+2. An Airbyte Engineer to follow the  [Connector Breaking Change Release Playbook](https://docs.google.com/document/u/0/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit) before merging.
+
+
+### Types of Breaking Change(s):
+A breaking change is any change that will require users to take action before they can continue to sync data. The following are examples of breaking changes:
```

Comment:
Can you link to the semantic versioning guide and make clear that breaking change == major bump?